### ISSUE TITLE
Map editor ground tiles not all loaded

### ISSUE THREAD

**Issue by gvissers** [raduprv/Eternal-Lands#79]
This is probably not very high on the priority list, but:

Does anyone find it strange that the map editor only load ground tiles already present in the map? If I want to add a new tile on a new map, I can select either water or grass, or I can blindly click on one of the squares containing whatever the map editor happens to load as texture 0. If I then save the map, and load it again, the newly selected tile comes up.

I guess the map editor should simply load all available ground tiles. In addition, it should maybe disallow selecting new tiles for which no texture can be found (though this might be problematic if one just wants to make a small change to a map but doesn't have up to date data file, so maybe not).

As said, not high priority, but it bugs me.

**Comment by feeltheburn** [comment]
I have noticed it. Annoying but I got around it by loading a trash map first that has every possible ground tile, then loading the map I'm working on. (Usually the map I'm working on already has the tiles I'd use so it's rare I have to use it.)

The editor will remember any ground tiles from previously opened maps so this works.

Here's the trash map if you want it for a temporary fix.

[ZZZZZZZ-GROUND-TILES.elm.gz](https://github.com/raduprv/Eternal-Lands/files/5360872/ZZZZZZZ-GROUND-TILES.elm.gz)
